scivision/io/reader.py

```diff
@@ -24,7 +24,7 @@ def _parse_config(path: os.PathLike) -> dict:
 
 
 def load_pretrained_model(
-    path: os.PathLike, allow_install: bool = False, *args, **kwargs
+    path: os.PathLike, branch: str = 'main', allow_install: bool = False, *args, **kwargs
```

Comment:
Seems like you could also add the config file here?

```suggestion
    path: os.PathLike, 
    branch: str = 'main', 
    config_filename: str = SCIVISION_YAML_CONFIG,
    allow_install: bool = False, 
    *args, 
    **kwargs
```